deploy/foxlake/templates/keda-addon.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions.kubeblocks.io/v1alpha1
+kind: Addon
+metadata:
+  name: keda
+
+spec:
+  type: Helm
+
+  helm:
+    chartLocationURL: https://kedacore.github.io/charts/keda-2.10.2.tgz
+
+  installable:
+    autoInstall: true
```

Comment:
having autoInstall=true, woud be better to have KEDA as sub-chart instead of addon, ref: https://helm.sh/docs/chart_template_guide/subcharts_and_globals/

Reply:
> having autoInstall=true, woud be better to have KEDA as sub-chart instead of addon, ref: https://helm.sh/docs/chart_template_guide/subcharts_and_globals/

Made this change in the latest commit.